### DISCUSSION TITLE
[FEATURE REQUEST] map in a source-block in org-mode

### DISCUSSION THREAD

**Discussion by ghost** [minad/osm#45] [Ideas]
Great package! I was wondering whether it is possible to get the map in a source-block or differently in a org-mode buffer. 
Like a picture preview, so it would be a geo-link preview (maybe not necessarily interactive).

Thanks!

**Comment by minad** [comment]
Nice idea, but difficult to realize. Static preview images are certainly doable. Note that Emacs does not support flowing text around images, so the result for embedded links would not be pretty. One could also consider embedding an entire map widget in Org. Unfortunately Emacs does not offer a facility to embed different modes into each other, so this would imply a large implementation effort.

**Reply by ghost** [reply]
ok, I would be happy with a static image as well. when clicking on it, it would open the map on the side.
